modules/Kconfig.nordic

```diff
@@ -3,6 +3,58 @@
 
 config HAS_NORDIC_DRIVERS
 	bool
+	default y if HAS_HW_NRF_RADIO_IEEE802154
+
+menu "Nordic drivers"
+	depends on HAS_NORDIC_DRIVERS
+
+menuconfig NRF_802154_RADIO_DRIVER
+	bool "Enable 802154 radio driver"
+	depends on HAS_HW_NRF_RADIO_IEEE802154
+
+if NRF_802154_RADIO_DRIVER
+
+choice NRF_802154_CCA_MODE
+	prompt "nRF52 IEEE 802.15.4 CCA mode"
```

Comment:
nRF53 also features IEEE 802.15.4 mode, so perhaps these prompts should be updated then. Though I'm not sure if it should be done in this PR.

Reply:
I'll simply make it `nRF` then.